tests/app/views/contexts/conftest.py

```diff
@@ -0,0 +1,53 @@
+import json
+from mock import MagicMock
+from pathlib import Path
+
+import pytest
+
+from app.data_model.answer import Answer
+from app.data_model.answer_store import AnswerStore
+from app.data_model.list_store import ListStore
+from app.forms.questionnaire_form import QuestionnaireForm
+from app.setup import create_app
+
+
+@pytest.fixture
+def rendered_block():
+    path = Path(__file__).parent.joinpath('test_list_collector_rendered_block.json')
+    with open(path, 'r') as fp:
+        return json.load(fp)
+
+
+@pytest.fixture
+def answer_store():
+    return AnswerStore(
+        [
+            {'answer_id': 'first-name', 'value': 'Toni', 'list_item_id': 'PlwgoG'},
```

Comment:
Is it worth adding another person here? We're testing a summary of only one person at the moment I think.